tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -57,104 +86,104 @@ describe.skip('CorrectNameRequest', () => {
     const wrapper = wrapperFactory()
 
     // Verify Invalid before input
-    expect(wrapper.vm.isFormValid).toBe(false)
+    expect(wrapper.vm.formValid).toBe(true) // default formValid is not false
 
-    wrapper.vm.nameRequestNumber = 'NR 1234567'
+    wrapper.vm.nrNumber = 'NR 1234567'
     wrapper.vm.applicantPhone = '123 456 7890'
     wrapper.vm.applicantEmail = 'mock@example.com'
 
     await flushPromises()
 
-    expect(wrapper.vm.isFormValid).toBe(true)
+    expect(wrapper.vm.formValid).toBe(true)
   })
 
   it('verifies invalid NR', async () => {
     const wrapper = wrapperFactory()
 
     // Verify Invalid before input
-    expect(wrapper.vm.isFormValid).toBe(false)
+    expect(wrapper.vm.formValid).toBe(true)
```

Comment:
Is the form valid before input? Please update comment on line 103.

Reply:
Also on lines 117, 132, 145, 161, 177, 219, etc.

Reply:
Yes. I will update